wakuv2/waku.go

```diff
@@ -1447,6 +1559,17 @@ func (w *Waku) processReceivedMessage(e *common.ReceivedMessage) {
 		w.storeMsgIDsMu.Unlock()
 	}
 
+	if e.MsgType == common.SendMessageType && !(*e.Envelope.Message().Ephemeral) {
```

Comment:
Isn't this the receivedMsgs function? 
Shouldn't this logic be in https://github.com/status-im/status-go/blob/94e5431ea311079cc5cdd0e185b646497e134a7f/wakuv2/waku.go#L986 which handles outgoing messages (for both LightPush and relay)?